.gitignore

```diff
@@ -46,3 +46,6 @@ gen/
 
 # agent build ignore
 /agent/
+
+# shardingsphere-example-generated ignore
+/examples/shardingsphere-sample/shardingsphere-example-generated/
```

Comment:
It is better to move un-commit's code to target folder.
It will make confuse if some src codes do not commit to GitHub.

Reply:
All right, i will check it and fix it in anthor pr.